docs/operations/setting_up_the_swarm.rst

```diff
@@ -170,3 +176,83 @@ files to the server.
      --entrypoints='Name:https Address::443 TLS:/etc/ssl/certs/openstax.crt,/etc/ssl/private/openstax.pem' \
      --logLevel=INFO \
      --accessLog
+
+Add cronjob to run docker prune
+===============================
+
+Docker swarm does not come with any kind of "garbage collection" for dangling 
+volumes or unused containers that are created when doing updates or after 
+restarts. This has caused issues where the host nodes run out of hard drive storage. To 
+prevent this we have created an :term:`Ansible` playbook to configure a cronjob on the server.
+
+Local or from bastion2?
+----------------------
+
+There are two places you can run this playbook.
+
+1. From ``localhost`` if you have ``bastion2`` setup as a :term:`JumpHost` and proper key 
+   to cops servers in the correct directory.
+2. From ``bastion2`` directly.
+
+See this `guide <https://www.tecmint.com/access-linux-server-using-a-jump-host>`_ 
+to learn how to configure a :term:`JumpHost`.
```

Comment:
This may be in a separate PR as I thought I remembered seeing it doc'd somewhere during your screen share session though can't find it currently: Should we include the ~/.ssh/config snippet example for this config in the docs somewhere if not already present?